bin/build-release-ccd-definition.sh

```diff
@@ -16,11 +15,16 @@ elif [ ${environment} == demo ]; then
 elif [ ${environment} == local ]; then
   # upload doesn't currently work with this command due to SDO and SDO-HNL files
   excludedFilenamePatterns="-e *-prod.json"
-elif [ ${excludeNonProdFiles} == true ]; then
-  excludedFilenamePatterns="-e UserProfile.json,*-nonprod.json,*-COS-nonprod.json,*GAspec.json,*-GAR2GAspec-nonprod.json,*DJ.json,*DJspec.json,*DJ-SDO-nonprod.json,*DJ-nonprod.json,*CUI.json,*CUI-nonprod.json,*-GAR3CP-nonprod.json"
+elif [ ${environment} == aat ]; then
+  excludedFilenamePatterns="-e UserProfile.json,*-nonprod.json,*GAspec.json,*DJ.json,*DJspec.json,*CUI.json"
```

Comment:
We are missing SDO related files here

Reply:
No, we're not.

If you notice on the left, the exclusion has `*-nonprod.json`, which would automatically include `*DJ-SDO-nonprod.json`.
The bit on the right it's a simplification, as all those matching `*-nonprod.json` were already excluded. If anything, this simplification should prevent mistakes.